test/snapshots.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
I guess you can try to also test the event handlers (at least some of then) by using shallow rendering and `.simulate`.